packages/quick_actions/quick_actions_ios/ios/Classes/PrivateHeaders/FLTQuickActionsPlugin_Test.h

```diff
@@ -7,7 +7,11 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// APIs exposed for unit tests.
-@interface FLTQuickActionsPlugin (Test)
+@interface FLTQuickActionsPlugin ()
+
+/// The type of the shortcut item selected when launching the app.
+/// API exposed for unit tests.
+@property(nonatomic, strong, nullable) NSString *launchingShortcutType;
```

Comment:
In next PR I will refactor `FLTQuickActionsPlugin` so that we won't have to expose these private APIs, as discussed in the proposal [here](https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o/edit?resourcekey=0-_cUjF1c0iBvRLKfV-3gK5A#heading=h.24hjfzhkz2dj).